_posts/platform/internals/2000-01-01-billing.md

```diff
@@ -4,10 +4,20 @@ modified_at: 2019-03-22 00:00:00
 tags: billing
 ---
 
+## Fill the Billing Profile
+
+In order to provision larger than Free-tiers resources, you have to fill your billing profile.
```

Comment:
This is not the only reason to fill a billing profile. It can be because the free trial is over.